apps/crn-server/test/helpers/teardown.ts

```diff
@@ -23,9 +23,11 @@ export const teardownHelper = (
   }
 
   return async () => {
-    for (const record of toDelete) {
-      await record.client.delete(record.id);
-    }
+    try {
+      for (const record of toDelete) {
+        await record.client.delete(record.id);
+      }
+    } catch {}
```

Comment:
Shouldn't the try-catch be inside the loop? This way it will stop iterating if it fails on any one deletion, but if you move the try-catch to inside the for-of then it will still (attempt to) delete all records.